cromwell.example.backends/cromwell.examples.conf

```
@@ -12,6 +19,16 @@ webservice {
 
 akka {
   # Optionally set / override any akka settings
+  http {
+    server {
+      # Increasing these timeouts allow rest api responses for very large jobs
+      # to be returned to the user. When the timeout is reached the server would respond
+      # `The server was not able to produce a timely response to your request.`
+      # https://gatkforums.broadinstitute.org/wdl/discussion/10209/retrieving-metadata-for-large-workflows
+      # request-timeout = 300s
```

I would prefer to list the default, which I believe is 20s

```suggestion
      # request-timeout = 20s
```